src/guidescan_web/genomics/resolver.clj

```diff
@@ -177,9 +177,9 @@
     (if-let [conns (get-in config [:config :sequence-resolvers])]
       (do
         (timbre/info "Checking sequence-resolvers")
-        (for [organism (keys conns)]
+        (doseq [organism (keys conns)]
           (if-not (resolve-sequence-raw (get-in conns [organism :url])
-                                        random-dna-seq)
+                                        (get-in conns [organism :seq] random-dna-seq))
```

Comment:
When the test sequence does work, because its completely random, it didn't actually run successfully, so I get the `"organism %s's search endpoint does not appear to be available"` message in any case. This should allow a sequence guaranteed to work for an organism to be tested, if available in the `config.edn`. We can add to that list later.